ai_serving/fasttext_clf/fasttext_clf_app.py

```diff
@@ -0,0 +1,49 @@
+from flask import request
+
+from base.ai_serving_base import AIServiceBase, FlaskError
+
+from train import load_model
+from predict import predict
+
+
+class FasttextClfBase:
+    model = None
+
+    def predict(self, sentences):
+        self.model = self.model or load_model()
```

Comment:
Is there a reason for loading (and potentially training the model) on first query, as opposed to loading it on initialziation at line 10?